<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushConnectButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDisConnectButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>DisConnect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>18</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PORT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_baud">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>46</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BAUD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Port">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Baud">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>831</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Msg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>629</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSendButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>630</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushTestButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TEST</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
